packages/suite/src/views/dashboard/components/AssetsCard/components/AssetGrid.tsx

```diff
@@ -119,8 +124,17 @@ export const AssetGrid = React.memo(({ network, failed, cryptoValue }: AssetGrid
 
                     <Coin>{name}</Coin>
                 </CoinNameWrapper>
-
-                {!isTestnet(symbol) && <Ticker symbol={symbol} />}
+                {!isTestnet(symbol) && (
+                    <>
+                        <TickerWrapper>
+                            <Ticker symbol={symbol} />
```

Comment:
I'd prefer adding a `className` prop to Ticker and styling it rather than adding a wrapper element.